ddtrace/tracer/tracer.go

```diff
@@ -357,12 +357,24 @@ const sampleRateMetricKey = "_sample_rate"
 
 // Sample samples a span with the internal sampler.
 func (t *tracer) sample(span *span) {
+	// Apply priority sampling
+	if t.config.prioritySampling != nil {
+		// Only apply if a priority hasn't been set yet.
+		// It may have been set by tags during initialization.
+		if !span.context.hasPriority {
+			t.config.prioritySampling.apply(span)
+		}
+		return
+	}
+
+	// TODO: Decide if this code and related tests should be removed.
 	sampler := t.config.sampler
 	sampled := sampler.Sample(span)
 	span.context.sampled = sampled
 	if !sampled {
 		return
 	}
+
```

Comment:
I don't think we need this empty line. I feel the closure of the `if` clause creates enough spacing for the code to be readable.